test/NuGet.Clients.Tests/NuGet.CommandLine.Test/CommandRunner.cs

```diff
@@ -29,6 +26,7 @@ public class CommandRunner
                 RedirectStandardError = true,
                 RedirectStandardInput = inputAction != null
             };
+            psi.EnvironmentVariables["NuGetTestModeEnabled"] = "True";
```

Comment:
Talk to zhi

Sent from my Windows Phone
________________________________
From: Xavier Decoster<mailto:notifications@github.com>
Sent: ‎12/‎16/‎2015 12:48 PM
To: NuGet/NuGet.Client<mailto:NuGet.Client@noreply.github.com>
Cc: Yishai Galatzer<mailto:yigalatz@microsoft.com>
Subject: Re: [NuGet.Client] when in test mode, the user agent string should be set to NuGet Test Client (#135)


In test/NuGet.Clients.Tests/NuGet.CommandLine.Test/CommandRunner.cs<https://github.com/NuGet/NuGet.Client/pull/135#discussion_r47828752>:

> @@ -29,6 +26,7 @@ public class CommandRunner
>                  RedirectStandardError = true,
>                  RedirectStandardInput = inputAction != null
>              };
> +            psi.EnvironmentVariables["NuGetTestModeEnabled"] = "True";


Also, there's this comment, not sure if taken care of: #135 (comment)<https://github.com/NuGet/NuGet.Client/pull/135#issuecomment-159367341>

—
Reply to this email directly or view it on GitHub<https://github.com/NuGet/NuGet.Client/pull/135/files#r47828752>.